TTD/TTD.hpp

```diff
@@ -83,6 +83,9 @@ namespace TTD {
 		struct TTD_Replay_IThreadView_vftable* IThreadView;
 	} TTD_Replay_IThreadView;
 
+	#define MEM_READ_EVENT_FLAG 0
```

Comment:
Is these constants meant to be used somewhere?

Reply:
Yes, I used them in my memCallback function:
```
bool memCallback(unsigned __int64 callback_value, TTD::TTD_Replay_MemoryWatchpointResult* mem, struct TTD::TTD_Replay_IThreadView* thread_info)
{
	//...
	if (mem->flags == MEM_READ_EVENT_FLAG)
	{
                //p_mem_content is a struct pointer to log single memread/write event information
		p_mem_content->m_b_read = true;
	}
	else
	{
		p_mem_content->m_b_read = false;
	}
	//...

	return false;
}
```
If you don't like them, just delete these constants. :)

Reply:
Nop, it's actually a valuable information.
It would be helpful to reference them in the corresponding structure's element, as a comment.
I've added a suggestion below.

If you prefer, I can rather add it in a future commit

Reply:
Thanks for you suggestion, this will make things much more clear.